### PR TITLE
update go client and zbctl with long polling timeout

### LINKED ISSUE #2931: Configure long polling timeout in go client and zbctl
**Description**

#2930 adds long polling timeout in grpc message for ActivateJobsRequest and makes it configurable from java client. We should also make it configurable in go client and zbctl.

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

make long polling timeout configurable in go client and zbctl

## Related issues

closes #2931

## Pull Request Checklist

- [x] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [x] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [x] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: clients/go/commands/activateJobs_command.go [hunk 1]
```diff
@@ -79,6 +81,12 @@ func (cmd *ActivateJobsCommand) FetchVariables(fetchVariables ...string) Activat
 	return cmd
 }
 
+func (cmd *ActivateJobsCommand) RequestTimeout(timeout time.Duration) ActivateJobsCommandStep3 {
```
Comment (MiguelPires) [inline clients/go/commands/activateJobs_command.go]: Why do we need this method if we already have to specify the timeout in NewActivateJobsCommand?

File: clients/go/worker/jobWorker_builder.go [hunk 2]
```diff
@@ -96,6 +99,12 @@ func (builder *JobWorkerBuilder) Timeout(timeout time.Duration) JobWorkerBuilder
 	return builder
 }
 
+func (builder *JobWorkerBuilder) RequestTimeout(timeout time.Duration) JobWorkerBuilderStep3 {
```
Comment (MiguelPires) [inline clients/go/worker/jobWorker_builder.go]: Same question as above

**Review by miguelpires — CHANGES_REQUESTED** [review]
Looks good, I just have a couple of questions.

File: clients/go/commands/activateJobs_command.go (hunk 1 above)
Comment (deepthidevaki) [reply clients/go/commands/activateJobs_command.go]: zbctl uses this method to configure the timeout.

Comment (MiguelPires) [reply clients/go/commands/activateJobs_command.go]: Ah I missed the 2nd commit, my bad

**Review by miguelpires — APPROVED** [review]
bors r+

**Pull request merged**